lib/new_roman_numerals.js

```diff
@@ -0,0 +1,15 @@
+function convertToNewRoman(num){
+  var arabicNums = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4 ,1]
+  var romanNums = ["M","CM","D", "CD","C", "XC","L","XL","X", "IX","V", "IV" ,"I"]
```

Comment:
we're limiting the flexibility of our solution to these two variables that are hardcoded, BUT i think that's perfectly fine because these are constants and aren't going to change. 

there may be a better data structure to use here rather than just two arrays. we could make something that keeps these two conversions together